### PR TITLE
Update capa explorer scopes in rule template

### PR DESCRIPTION
**Pull request by mr-tz**
### Checklist

- [x] No CHANGELOG update needed

- [ ] No new tests needed

- [ ] No documentation update needed

### CODE REVIEW

File: capa/ida/plugin/view.py [hunk 1]
```diff
@@ -194,7 +194,9 @@ def load_preview_meta(self, ea, author, scope):
             "    namespace: <insert_namespace>",
             "    authors:",
             f"      - {author}",
-            f"    scope: {scope}",
+            f"    scopes:",
```
Comment (mr-tz) [inline capa/ida/plugin/view.py:197]:
```suggestion
            "    scopes:",
```

File: capa/ida/plugin/view.py [hunk 2]
```diff
@@ -194,7 +194,9 @@ def load_preview_meta(self, ea, author, scope):
             "    namespace: <insert_namespace>",
             "    authors:",
             f"      - {author}",
-            f"    scope: {scope}",
+            "    scopes:",
+            f"      static: {scope}",
+            "      dynamic: unspecified",
```
Comment (mike-hunhoff) [inline capa/ida/plugin/view.py:199]:
```suggestion
            "    scopes:",
            f"      static: {scope}",
            "       dynamic: unspecified",
```

Comment (mr-tz) [reply capa/ida/plugin/view.py:199]: note the f-string indent above, so this is correct as is

**Review by mike-hunhoff — APPROVED** [review]
Ran a quick test locally, looks good. LGTM 🚀

**Pull request merged**